app/views/taxons/sections/_news_and_communications.html.erb

```diff
@@ -5,7 +5,10 @@
 
 <div class="grid-row">
   <div class="taxon-page__featured-item column-one-third <%= featured_item_layout_class %>" <%= "data-module=track-click" if track_click?(section[:promoted_content]) %>>
-    <img class="taxon-page__featured-image" src="<%= featured_item[:image][:url] %>" alt="<%= featured_item[:image][:alt] %>">
+    <%= render "govuk_publishing_components/components/image_card", {
+        href: featured_item[:link].fetch(:path),
+        image_src: featured_item[:image][:url]
+    } %>
     <div class="taxon-page__featured-text">
```

Comment:
If we render the heading and metadata with the component, we can then remove this and any associated `taxon-page__featured` CSS